cellfinder/__init__.py

```diff
@@ -1,25 +1,50 @@
+import os
+import warnings
 from importlib.metadata import PackageNotFoundError, version
 from pathlib import Path
 
+# Check cellfinder is installed
 try:
     __version__ = version("cellfinder")
 except PackageNotFoundError as e:
     raise PackageNotFoundError("cellfinder package not installed") from e
 
-# If tensorflow is not present, tools cannot be used.
+# If Keras is not present, tools cannot be used.
 # Throw an error in this case to prevent invocation of functions.
 try:
-    TF_VERSION = version("tensorflow")
+    KERAS_VERSION = version("keras")
 except PackageNotFoundError as e:
+    raise PackageNotFoundError(
+        f"cellfinder tools cannot be invoked without Keras. "
+        f"Please install Keras with a backend into your environment "
+        f"to use cellfinder tools. "
+        f"For more information on Keras backends, please see "
+        f"https://keras.io/getting_started/#installing-keras-3."
+        f"For more information on brainglobe, please see "
+        f"https://github.com/brainglobe/brainglobe-meta#readme."
+    ) from e
+
+
+# If no backend is configured and installed for Keras, tools cannot be used
+# Check if backend is configured. If not, set to "torch"
+if not os.getenv("KERAS_BACKEND"):
+    os.environ["KERAS_BACKEND"] = "torch"
+    warnings.warn("Keras backend not configured, automatically set to torch")
```

Comment:
I wouldn't have this warning by default. Every user will see this warning, but they won't actually need to do anything about it.

Reply:
Done